apis/types/capability.go

```diff
@@ -206,14 +206,15 @@ type Addon struct {
 
 // AddonMeta defines the format for a single addon
 type AddonMeta struct {
-	Name         string             `json:"name" validate:"required"`
-	Version      string             `json:"version"`
-	Description  string             `json:"description"`
-	Icon         string             `json:"icon"`
-	URL          string             `json:"url,omitempty"`
-	Tags         []string           `json:"tags,omitempty"`
-	DeployTo     *AddonDeployTo     `json:"deployTo,omitempty"`
-	Dependencies []*AddonDependency `json:"dependencies,omitempty"`
+	Name          string             `json:"name" validate:"required"`
+	Version       string             `json:"version"`
+	Description   string             `json:"description"`
+	Icon          string             `json:"icon"`
+	URL           string             `json:"url,omitempty"`
+	Tags          []string           `json:"tags,omitempty"`
+	DeployTo      *AddonDeployTo     `json:"deployTo,omitempty"`
+	Dependencies  []*AddonDependency `json:"dependencies,omitempty"`
+	NeedNamespace []string           `json:"needNamespace,omitempty"`
```

Comment:
What does this field mean?

Reply:
This will be the name of the Namespace which will be created.